chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/typingIndicatorMiddleware.tsx

```diff
@@ -32,7 +34,7 @@ const TypingIndicator = ({ activeTyping, visible }: any) => {
         if (activeTyping[i].role && activeTyping[i].role === DirectLineSenderRole.User) {
             //visible is set to false if the current user is typing, in which case, we just send typing indicator to OC
             if (state.domainStates.liveChatConfig?.LiveChatVersion === 2 && !visible) {
-                chatSDK?.sendTypingEvent();
+                debounceTyping();
```

Comment:
Is the debounce needed now that we've already enabled `sendTypingIndicator` on WebChat?